plugins/scanners/scanoss/src/main/kotlin/ScanOss.kt

```diff
@@ -65,8 +68,29 @@ class ScanOss(
     override fun scanPath(path: File, context: ScanContext): ScanSummary {
         val startTime = Instant.now()
 
+        val basePath = path.toPath()
+
+        val filterConfig = FilterConfig.builder()
+            .customFilter { currentPath ->
+                // The "currentPath" variable contains a path object representing the file or directory being evaluated
+                // by the filter.
+                // This is provided by the Scanner and represents individual files/directories during traversal.
+                try {
+                    val relativePath = basePath.relativize(currentPath).toString()
```

Comment:
Prefer `currentPath.toFile().toRelativeString(path)`, which allows you to drop `val basePath = path.toPath()` above.